### ISSUE TITLE
Backdrop CSS issues

### ISSUE THREAD

**Issue by laryn** [civicrm/civicrm-backdrop#84]
I just rebuilt my local and see some issues with default styling out of the box. Noting here to come back to at some point:

1) CiviCRM News on CiviCRM dashboard:
![Screen Shot 2019-06-25 at 5 18 25 PM](https://user-images.githubusercontent.com/871421/60137729-7fa10800-976d-11e9-984d-7e5745625b69.jpg)

2) The CiviCRM icon isn't showing up consistently in the admin menu ([related](https://github.com/civicrm/civicrm-core/commit/d561265f8cb1d93f9521c8eafc14037a3a826cec)):
![Screen Shot 2019-06-25 at 5 20 43 PM](https://user-images.githubusercontent.com/871421/60137762-9a737c80-976d-11e9-996b-8634d9711008.jpg)

**Comment by laryn** [comment]
For the first point, the Backdrop CSS is overriding/inserting the blue background. We can reset that override that for CRM content with the following for Backdrop:

```
.crm-container .ui-state-default, 
.crm-container .ui-widget-content .ui-state-default, 
.crm-container .ui-widget-header .ui-state-default {
  background: #e6e6e6 url(/modules/civicrm/bower_components/jquery-ui/themes/smoothness/images/ui-bg_glass_75_e6e6e6_1x400.png) 50% 50% repeat-x;
}
```

Should this go in the core `backdrop.css` file, or in a Backdrop-module css file (see below for more about that potential file).

Regarding the second point above, should I move [the bit of CSS that is logo-related from the core `backdrop.css` file](https://github.com/civicrm/civicrm-core/commit/d561265f8cb1d93f9521c8eafc14037a3a826cec) and into CSS that this module inserts instead? Right now it only shows up when on a CiviCRM page, but we actually want that bit to be added even to the Backdrop pages (so the CiviCRM icon shows up on all pages).

**Comment by laryn** [comment]
Yes, typically that's right, but this is how it comes "out of the box" at least with `civibuild` and this is how it displays on the Backdrop demo that the public uses to get their first impression of Backdrop and CiviCRM together. Would be nice to make that look cleaner!

**Comment by herbdool** [comment]
Maybe we want both. `civibuild` should set the admin theme and give the demo user permissions to see it. And we could also fix that for cases where people don't have access to the admin theme on their own installs.

(Seems a bit antiquated that there's a permission to admin theme. What's so dangerous about a theme?)

**Comment by laryn** [comment]
Just checked on the demo site and this has helped, but there are still some styling issues with menus and z-index, by the looks of things:

![Screen Shot 2019-12-11 at 9 32 06 AM](https://user-images.githubusercontent.com/871421/70635318-511ec880-1bf9-11ea-8e6c-8628bba527bb.jpg)

(Aside: It would be good to have the demo set the theme to Seven and choose a full width Layout as well.)

**Comment by herbdool** [comment]
My guess is that the demo's build process needs to add a permission to view the admin theme in order for it to show up. That might be main thing.

Odd that z-index thing. Wonder why I haven't seen that before.

**Comment by laryn** [comment]
> Odd that z-index thing. Wonder why I haven't seen that before.

Probably because you don't use Basis as a CiviCRM theme. 😊
